tests/unit/test_utilities.py

```diff
@@ -79,7 +80,9 @@ def test_get_variable_values(self):
                 wind_speed_values = get_variable_values(dataset, wind_speed, None)
                 self.assertIsInstance(wind_speed_values, np.ndarray)
                 self.assertEqual(len(wind_speed_values.shape), 2)
-                self.assertEqual(wind_speed_values.shape, wind_speed.shape)
+                self.assertEqual(
+                    wind_speed_values.shape, np.ma.transpose(wind_speed).shape
```

Comment:
I don't doubt this is correct, so for my own learning's sake, how is `wind_speed_values.shape` == `np.ma.transpose(wind_speed).shape`?